### PR TITLE
Translate 2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md

### PR DESCRIPTION
**Pull request by yixin21**
zh-trans 2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md

ref:k8smeetup/website-tasks#3363

### CODE REVIEW

File: content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+<!--
+The second [pull-request cleaned up an unused import](https://github.com/kubernetes/kubernetes/pull/22321) in the Daemonset unit test file. The import statement broke the unit tests with golang 1.4. Our nightly Jenkins job helped us find this error and we promptly pushed a fix for it.  
+-->
+第二个 [清理未使用导入的请求](https://github.com/kubernetes/kubernetes/pull/22321) 在 Daemonset 单元测试文件中。import 语句打破了 golang 1.4 的单元测试。我们的夜间 Jenkins 工作帮助我们找到错误并且迅速完成修复。
+
+<!--
+**CONCLUSION AND FUTURE WORK**  
+-->
+**结论于未来工作**
```
Comment (tengqm) [inline content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md:100]: typo

File: content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+-->
+联合测试文档更详细地描述了这些步骤。运行端到端测试并上传结果的脚本简化了贡献新联合测试作业的工作量。设置自动化测试作业的具体步骤以及在其中部署 Kubernetes 的合适环境将留给读者进行选择。对于使用 Jenkins 的组织，用于 GCE 和 GKE 测试的 jenkins-job-builder 配置可能会提供有用的示例。
+
+
+<!--
+**RETROSPECTIVE**  
+
+The e2e tests on Azure have been running for several weeks now. During this period, we have found two issues in Kubernetes. Weixu Zhuang immediately published fixes that have been merged into the Kubernetes master branch.  
+-->
+**追溯**
```
Comment (tengqm) [inline content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md:83]: 回顾

Comment (yixin21) [inline content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md:83]:
```suggestion
**回顾**
```

File: content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md (hunk 1 above)
Comment (yixin21) [inline content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md:100]:
```suggestion
**结论与未来工作**  
```

File: content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md (hunk 2 above)
Comment (yixin21) [reply content/zh/blog/_posts/2016-07-00-Bringing-End-To-End-Kubernetes-Testing-To-Azure-2.md]: 已订正

**Comment by tengqm** [comment]
Please squash you commits into one.

**Comment by yixin21** [comment]
> Please squash you commits into one.

So sorry I don't know how to squash, would  you please help?

**Comment by xichengliudui** [comment]
git rebase -i HEAD~3
@yixin21

**Pull request merged**